### PR TITLE
Set a list of architectures that snap package runs on

### PR DESCRIPTION
**Pull request by ilya-fedin**

### CODE REVIEW

**Comment by ilya-fedin** [comment]
Looks like a bug in snapcraft :see_no_evil:

**Comment by ilya-fedin** [comment]
Okay, then without `build-error: ignore`

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -6,6 +6,13 @@ base: core18
 grade: stable
 confinement: strict
 
+architectures:
+  - build-on: amd64
+  - build-on: i386
+  - build-on: arm64
+  - build-on: armhf
+  - build-on: ppc64el
```
Comment (23rd) [inline snap/snapcraft.yaml]: Will it work this way?
```suggestion
architectures: [amd64, i386, arm64, armhf, ppc64el]
```

Comment (ilya-fedin) [reply snap/snapcraft.yaml]: No. Look at example 6:
https://snapcraft.io/docs/architectures

Comment (23rd) [reply snap/snapcraft.yaml]: Okay.

**Comment by john-preston** [comment]
Thanks!

**Pull request merged**